velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -491,6 +500,31 @@ std::vector<RowVectorPtr> AggregationFuzzer::generateInputData(
   return input;
 }
 
+std::vector<RowVectorPtr> AggregationFuzzer::generateInputDataWithId(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  names.push_back("id");
+  types.push_back(BIGINT());
+  auto inputType = ROW(std::move(names), std::move(types));
+
+  std::vector<RowVectorPtr> input;
+  auto size = vectorFuzzer_.getOptions().vectorSize;
+  int64_t identifier = 0;
+  for (auto i = 0; i < FLAGS_num_batches; ++i) {
+    std::vector<VectorPtr> children;
+    for (auto i = 0; i + 1 < inputType->size(); ++i) {
```

Comment:
nit: i < inputType->size() - 1 for readability

Reply:
"+1" on the left side is actually needed to avoid unsigned integer underflow because inputType->size() is a uint32_t. (inputType->size() usually should not be 0 though, but just in case.)

Reply:
Can we add VELOX_CHECK_GT(inputType->size(), 0) to be sure ?

Reply:
After the change, I found that we can be sure that types.size() is at least one because we manually inserted BIGINT() to it. So I just moved "+1" on the left side to "-1" on the right side.